### ISSUE TITLE
Support a better naming_scheme for a forked repository

### ISSUE THREAD

**Issue by Chumper** [renovatebot/renovate#7128]
When running Renovate in `forkMode: true`, Renovate will fork a repository to its personal namespace first before creating PRs from it. 

When there are multiple repositories with the same name Renovate gets confused because the fork in the personal space now serves two different repositories.

I would like to have the option for Renovate to rename the fork to `${org}_${repo}` o.s. so that we do not run into this kind of problems.

**Did you already have any implementation ideas?**

I would like to have an option `forkRename` for the config and when set do a rename after a fork is created.

Alternatively this can be a template but I think true/false is easier atm.

**Are there any workarounds or alternative ideas you've tried to avoid needing this feature?**

**Is this a feature you'd be interested in implementing yourself?**

**Comment by rarkins** [comment]
I hadn't expected that it should "get confused", and looking at the code then I'm confused if that's so. In this line is specifically forks the parent repo into `config.repository`: https://github.com/renovatebot/renovate/blob/21e1f1ff3edf975a26a4f2d846bc47bdad5ce5fd/lib/platform/github/index.ts#L330-L337

So the only way there'd be a problem is if GitHub returns the wrong repo to that command, which seems unlikely?

**Comment by Chumper** [comment]
You are correct, I created two test orgs:
https://github.com/renovate-test-1/fork-test
https://github.com/renovate-test-2/fork-test

And when Renovate runs it will fork them to:
https://github.com/Chumper/fork-test
https://github.com/Chumper/fork-test-1

So there is already some name changing going on. 
My problem was that I deleted one fork which caused some PRs to be from an `unknown repository`.

I am closing this.
